blocks/extras/extensions/plot.py

```diff
@@ -157,9 +166,39 @@ def preexec_fn():
     def __getstate__(self):
         state = self.__dict__.copy()
         state['sub'] = None
+        state['_push_thread'] = None
```

Comment:
I think what you want is `del state['_push_thread']`. Otherwise the `hasattr` call on line 125 will return `True` and `None` will be returned on an unpickled object.